packages/fxa-payments-server/src/components/Header/index.scss

```diff
@@ -0,0 +1,49 @@
+@import '../../../../fxa-content-server/app/styles/breakpoints';
+
+#fxa-settings-header {
```

Comment:
Do you have a SCSS linter installed? I have `scss-lint` installed from the VSCode extensions list (perhaps there's a better one), most of these squiggles are due to indentation but here's what I'm seeing:

<img width="400" alt="image" src="https://user-images.githubusercontent.com/13018240/75078200-9f3e2b00-54ca-11ea-9fa9-80f2a1401867.png">

Another thing I'm planning to address in the content server with the settings redesign is fixing up all of our warnings. One major player in this is styling with IDs, it's best practice to style with classes or `.class-name [element]` and the SCSS linter is complaining about this. However... we could end up turning this SCSS rule off if the team strongly prefers to style with IDs, I haven't asked anyone yet what their preference is. So it'd be better practice to turn this into a class, but you could also make the argument that we haven't set a hard rule for this yet and I'm nitpicking too hard here when I'm going to revisit a lot of our CSS in a few months, heh.

At the very least, we can pull `#fxa-manage-account` out of `#fxa-settings-header` nesting. When styling with an ID you should be able to do that at the top-level since nothing else will be named `#fxa-settings-header` on the page to cause conflicting styles and thus this level of specificity.

Reply:
I also don't like styling with ids, I copied this style from another location in the repo. I'll update and fix linter warnings. I should be using the same linter configs as you, I'll see if it got disabled for some reason cus I haven't been seeing sass lint warnings for a while